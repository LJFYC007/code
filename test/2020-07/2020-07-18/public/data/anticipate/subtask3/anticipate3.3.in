3
50 -1000000 1000000
-976173 -939346 -900355 -870816 -847218 -826241 -792607 -705706 -704561 -700969 -685162 -657396 -645943 -625614 -545951 -523942 -513406 -494638 -365647 -354805 -243508 -35102 -6184 10740 25225 44931 50128 104896 110085 206106 306049 332466 431524 449245 466828 558260 589473 602005 646398 655404 694434 738322 745355 764971 871822 876374 912978 913298 960198 993304 
17518 111148 1639 48522 81418 4200 123534 555504 879186 860100 546281 874634 756030 985973 203849 488008 99340 5085 36054 26712 280504 121269 449905 65292 75197 19294 90411 94701 605259 45865 86982 987615 63497 916997 64939 30006 86605 88137 16683 786330 800665 53538 17403 59574 63641 816990 512554 30374 662491 655737 
