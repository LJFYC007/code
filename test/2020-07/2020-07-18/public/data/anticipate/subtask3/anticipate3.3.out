0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.327427864618
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.078567744935
0.000000000000
0.100969044990
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.064849621096
0.000000000000
0.000000000000
0.143370281213
0.000000000000
0.103527331840
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.024497752771
0.088217989638
0.000000000000
0.000000000000
0.000000000000
0.000000000000
0.068572368899
0.000000000000
0.000000000000
0.000000000000
0.000000000000
